openqaoa/optimizers/training_vqa.py

```diff
@@ -630,6 +637,15 @@ def optimize(self):
         elif self.method == 'spsa':
             print("Warning : SPSA is an experimental feature.")
             method = om.SPSA
+        elif self.method.lower().split('_')[0] == 'pennylane': # check if we are using a pennylane optimizer
+            method = ompl.pennylane_optimizer
+
+            self.options['method'] = self.method.lower().replace("pennylane_", "") 
```

Comment:
If the words `pennylane_` is deleted here, wouldn't future calls to `self.options['method']` return the wrong (i.e not pennylane) string?

Reply:
Any comments on this?

Reply:
OK, looks good to me